web/script/index.js

```diff
@@ -2,7 +2,10 @@ fetch('https://api.github.com/repos/R2Northstar/Northstar/releases/latest')
     .then(response => response.json())
     .then(data => {
         document.querySelectorAll('.download').forEach(d => { d.href = data.assets[0].browser_download_url });
-        console.log("Successfully fetched latest release download URL. Updating download links.")
+        console.log("Successfully fetched latest release download URL. Updating download links.");
+        var filesize = data.assets[0].size;
+        var sizeinmb = filesize/1024/1024;
+        document.querySelectorAll('.big-button.download > span').forEach(e => { e.innerHTML = 'DOWNLOAD ('+sizeinmb.toFixed(1)+' MB)'});
```

Comment:
innerText is safer so should be used even though we trust github to not tell us the file size is some arbitrary html string

Reply:
I forgot about that, I just copied what I had from discord last night.

Reply:
Changed to innerText